### DISCUSSION TITLE
t0 function?

### DISCUSSION THREAD

**Discussion by TripleFJeff** [gopro/labs#928] [Q&A]
I'm making a change to a saved QR, and as a verification that I don't screw anything up I'm trying to recreate the existing one using the QR Creator and selecting all the settings. I had pretty good notes on what was in this QR, but I can't seem to figure out what the "t0" is for in the saved one. 

This is the full string:

mVr1080p60fSoSM25,50t0e1hS0aS

mV - Video Mode
r1080p - 1080p
60 - 60fps
fS - SuperView
o - start of option identifier
SM25,50 - spot metering
t0 - ?????????
e1 - Hypersmooth On
hS0 - Hindsight Off
aS - Wind Reduction Off

**Comment by dnewman-gpsw** [comment]
It is used on older cameras that had to turn Protune on or off.  t0 is Protune off.  You can completely remove it on H9 or greater.
